client/dashboard/task-list/index.js

```diff
@@ -256,6 +256,58 @@ class TaskDashboard extends Component {
 		);
 	}
 
+	onSkipStoreSetup = () => {
+		const completedTasksNames = this.getTasks()
+			.filter( x => x.completed )
+			.map( x => x.title );
+
+		recordEvent( 'wcadmin_tasklist_skip', {
+			completed_tasks: completedTasksNames.length,
```

Comment:
I think to keep things a bit more semantic we can rename this to `completed_tasks_count` and `complete_tasks_names` to `complete_tasks` if you don't have any issue with it @pmcpinto.